### DISCUSSION TITLE
How can I get the latest tweet for a specific keyword and the recipient ID of the OP?

### DISCUSSION THREAD

**Discussion by 621625** [tweepy/tweepy#1649] [Questions] [answered]
How can I get the latest tweet for a specific keyword and the recipient ID of the OP?

Example keyword: 

blacksmith

I want just one tweet, the latest one with the keyword blacksmith as output. I also want the recipient ID (the ID of the person that posted it). Anyone has a snippet for this? I'd rather not use Discord to ask about it, I don't like Discord at all.

**Comment by Harmon758** [comment, marked as answer]
Duplicate of #1632 / #1637, where I've already answered your question

Also, you probably mean "author" or "user", not "recipient".
